modules/ROOT/pages/mule-core-usage-self.adoc

```diff
@@ -0,0 +1,20 @@
+= Monitor Mule Core Usage With Runtime Fabric on Self-Managed Kubernetes
+
+Anypoint Runtime Fabric on Self-Managed Kubernetes enables you to use your own infrastructure monitoring solution to view metrics, monitor and alert on conditions.
+YYour monitoring solution can also be used to count and generate alerts on Mule core usage.
```

Comment:
Typo: Your*

Reply:
Fixed; please re-review when able to allow for merge.